src/main/java/com/shapesecurity/salvation/Parser.java

```diff
@@ -252,6 +252,8 @@ private boolean eat(@Nonnull Class<? extends Token> c) {
                     if (mediaTypes.isEmpty()) {
                         this.error(token, "The media-type-list must contain at least one media-type.");
                         throw INVALID_MEDIA_TYPE_LIST;
+                    } else if (mediaTypes.stream().anyMatch(x -> x.matchesTypeOrSubType("*"))) {
+                        this.warn(token,"`*` is allowed character in media type which will be literally matched. Make sure it not an attempt to match any.");
```

Comment:
> Make sure it not an attempt to match any.

Missing an "is" in there. Also if the user/dev does want to define "any" what should they be using? Or is it not possible? Either way, some clarity or a suggestion might help...